ISX/ISX/LoginViewController.swift

```diff
@@ -20,6 +20,8 @@ class LoginViewController:UIViewController {
         super.viewDidAppear(animated)
         emailTextField.text = "cabin@crew.nl"
         passwordTextField.text = "cabincrew"
+        Database.database().isPersistenceEnabled = true
```

Comment:
als het goed is, is dit niet nodig, als persistence aanstaat in app delegate, is het genoeg???